### ISSUE TITLE
Not able to inspect the element

### ISSUE THREAD

**Issue by sahaiswat** [microsoft/WinAppDriver#1531]
There is an user info icon which on mouse hovering displays email and pin but through the Inspector the element does not get recognized. However, when I expand it and inspect other elements I can very well do that. So I am stuck at a point where I can't move ahead without clicking the element which is giving hard time being recognized. Can anyone please help. Below is the screenshot of the icon when clicked manually.

![image](https://user-images.githubusercontent.com/61317806/119755582-5c9da680-bef6-11eb-9903-aa4db081825e.png)

And this is the icon on mouse hovering

![image](https://user-images.githubusercontent.com/61317806/119755759-a1c1d880-bef6-11eb-93ea-518d49eed20e.png)
 
This is recorded through UI recorder for the other element which displays after the user icon is expanded.

"/Pane[@ClassName=\"#32769\"][@Name=\"Desktop 1\"]/Pane[@ClassName=\"Chrome_WidgetWin_1\"][@Name=\"xyz\"]/Document[@ClassName=\"Chrome_RenderWidgetHostHWND\"][@Name=\"xyz\"]/Custom/Custom[@AutomationId=\"window\"]/Custom/Custom[@AutomationId=\"userInfo\"]/Custom[@AutomationId=\"resetPinPopup\"]/Custom[@AutomationId=\"switchRegion\"]/Text[@Name=\"Switch Location\"]"

**Comment by sahaiswat** [comment]
Can you please elaborate? OCR? I need to click on this to go to the next element. Is this possible through OCR?

**Comment by anunay1** [comment]
Click on what email and pin?

**Comment by sahaiswat** [comment]
Yes. Because the flow needs the other elements to be clicked which can be
only done once email and pin is clicked

**Comment by anunay1** [comment]
You can use the actions class like below: 
 new Actions(session).moveToElement(element, 0, 0).moveByOffset(dx, dy).click().perform();

**Comment by sahaiswat** [comment]
But how to get the location of the element.

**Comment by anunay1** [comment]
You have have an automation id for userinfo. then what's the problem.

**Comment by Shakevg** [comment]
@sahaiswat Your c# code will be the next:
```
// find userInfo icon control 
var element = session.FindElement(Locator for userInfo icon);

// it is x coordinates of text to click from the left top corner of userInfo icon - maybe you need to adjust the value if it will not work
var dx = -90;
 // it is y coordinates of text to click from the left top corner of userInfo icon - maybe you need to adjust the value if it will not work
var dy = -90;

// click by related coordinates
new Actions(session).moveToElement(element, 0, 0).moveByOffset(dx, dy).click().perform();
```

**Comment by sahaiswat** [comment]
Thanks but this did not work. 
An element could not be located on the page using the given search parameters.
  Stack Trace: 
    RemoteWebDriver.UnpackAndThrowOnError(Response errorResponse)
    RemoteWebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
    AppiumDriver`1.Execute(String driverCommandToExecute, Dictionary`2 parameters)
    RemoteWebDriver.FindElement(String mechanism, String value)
    RemoteWebDriver.FindElementByXPath(String xpath)
    AppiumDriver`1.FindElementByXPath(String xpath)

**Comment by anunay1** [comment]
Can you share the code?

**Comment by sahaiswat** [comment]
public WindowsElement usericon => windriver.FindElementByXPath("//*[@AutomationId='userInfo']"); 

   

        public void ClickUserInfo()
        {
           
            Actions action = new Actions(windriver);
                     
            var dx = -110;
           
            var dy = -110;

            action.MoveToElement(usericon,0,0).MoveByOffset(dx, dy).Click().Perform();
            

           }

**Comment by anunay1** [comment]
The code is ok. Are you able to hover to the userinfo icon. 
Also can you share the page source before hover and after hover.

**Comment by Shakevg** [comment]
@sahaiswat Looks like that your xpath is not correct "//*[@AutomationId='userInfo']" or need to switch to new window https://github.com/microsoft/WinAppDriver/issues/1528#issuecomment-848187395.
Could you save page source and check that icon control really presents (use driver.PageSource) and then check that xpath working using http://xpather.com/?
Also, is control available in Inspector.exe?

**Comment by anunay1** [comment]
Can you also try with any other locator besides xpath?
